boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuflpr_defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_USE_DT_CODE_PARTITION=y
+
+# Execute from SRAM
+CONFIG_XIP=n
+
+CONFIG_DEBUG=y
+
+CONFIG_NRF_GRTC_TIMER=y
+
+CONFIG_PINCTRL=y
+
+# Enable GPIO
+CONFIG_GPIO=y
```

Comment:
And this could be moved to the common `nrf54l15pdk_defconfig`.

Reply:
This file is missing after PR #71149.